returnn/training.py

```diff
@@ -376,6 +367,24 @@ def create_returnn_config(
         if returnn_config.post_config is not None:
             post_config.update(copy.deepcopy(returnn_config.post_config))
 
+        if keep_epochs:
+            if not "cleanup_old_models" in post_config or isinstance(
+                post_config["cleanup_old_models"], bool
+            ):
+                assert (
+                    post_config.get("cleanup_old_models", True) == True
+                ), "'cleanup_old_models' can not be False if 'keep_epochs' is specified"
```

Comment:
Mh.. If `cleanup_old_models=False` then all epochs are kept, especially also the ones defined in `keep_epochs`. Maybe in this case we could ignore `keep_epochs` and leave it to `False`?

Just opening the discussion, I don have a strong opinion here.

Reply:
But wouldn't this mean that there are some settings that the user probably did not intend to have? But yes, ignoring would be an option...

Reply:
Yes, this would probably indicate a configuration error and we should alert the user with a failing assertion.